VERSION

```diff
@@ -1 +1 @@
-1.2.1
+1.3.0
```

Comment:
yes, all my projects follow semantic versioning. So new functionality, but without breaking old functionality is minor increase.